### ISSUE TITLE
C.S.S Wavebreaker won't TP me to the scene after I crashed.

### ISSUE THREAD

**Issue by KnightlyGlade** [Wynncraft/Issues#7675]
Short, basic story: The first time I got on the boat, it TPed me, but my wifi crashed and I got back on the server. Only problem was none of the dialogue was presenting and others got to TP. So I had to /kill to escape. Moments later I got back, but the boat won't TP me again. I tried these solutions.

/Class, Nothing happened.
Hopping around other servers, didn't work.

As of now, I'm standing where the tripwires (that supposed to TP you to the other scene) are, doing absolutely nothing but clicking noises.

Image: http://imgur.com/h06o3Wx

**Comment by KnightlyGlade** [comment]
Looks like you were mislead or didn't really mind the whole report.

**Comment by LordHelmchen98** [comment]
Did you manage to fix it?

**Comment by KnightlyGlade** [comment]
Nope. Every solution I tried, nothing happened. I travelled to Corkus by regular boat and the wavebreaker was able to take me back to Selchar. Unfortunately, it feels like I broke the quest, so I'm gonna have to wait...

**Comment by Moukassi** [comment]
I have the same problem
